sourced/ml/utils/engine.py

```diff
@@ -24,7 +24,10 @@ class EngineDefault:
     Default arguments for create_engine function and __main__
     """
     BBLFSH = "localhost"
-    VERSION = get_distribution("sourced-engine").version
+    try:
+        VERSION = get_distribution("sourced-engine").version
+    except DistributionNotFound:
+        VERSION = "0.5.7"
```

Comment:
Would be nice to show a warning in the logs. Also query the version from GitHub API for releases (`import requests`)

Reply:
done